docs/experiments.rst

```diff
@@ -103,6 +115,14 @@ as follows:
 The ``@INSTANCE:0@`` variable will resolve to ``/instance_directory/files[0]``, where ``files[0]`` is
 the first filename of the ``files`` key. Analogously for the ``@INSTANCE:1`` variable.
 
+If the instances have :ref:`InstanceExtraArguments`, we further need to add the :ref:`@-variable <AtVariables>`
+``@EXTRA_ARGS@`` to the experiment arguments, e.g,
+``args: ['./algorithm.py', '@INSTANCE:0', '@INSTANCE:1@', '@EXTRA_ARGS@']`` for the example above.
+``@EXTRA_ARGS@`` will resolve to the specified extra arguments of the respective instance (and also the used
+:ref:`variants <ExperimentsWithVariants>`, see below).
```

Comment:
I don't link that this paragraph is repeated three times. Can we shorten the repetitions and refer back to the first occurrence?

Reply:
I removed the repetition and references the first occurence